poly-commitment/src/lib.rs

```diff
@@ -21,7 +20,7 @@ use ark_poly::{
 use mina_poseidon::FqSponge;
 use rand_core::{CryptoRng, RngCore};
 
-pub trait SRS<G: CommitmentCurve>: Clone {
+pub trait SRS<G: CommitmentCurve>: Clone + Sized {
```

Comment:
Because `Sized` is required for the folding library to pass it in the type, IIRC.